_posts/2021-01-13-an-introduction-to-mixing-audio.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: post
+title:  "Introduction to Mixing Audio"
+author: flavius
+categories: [ audio ]
+image: ""
+featured: false
+---
+What is audio mixing? Is it helpful or a hindrance? What are my options for mixing audio?
```

Comment:
I almost want to see the tagline be, "Introduction to Mixing Audio - How to not sound like a Dial Up Modem". 🤣

Reply:
Your wish has come true :D